WindSpigot-Server/src/main/java/net/minecraft/server/EntityTrackerEntry.java

```diff
@@ -240,11 +245,12 @@ private boolean withinNoTrack(Entity entity) {
 
 		@Override
 		public void accept(EntityPlayer entityPlayer) {
-			if (!WindSpigotConfig.disableTracking || tracker.passenger == entityPlayer) {
+			synchronized (entityPlayer) { 
 				updatePlayer(entityPlayer);
```

Comment:
I believe this should check if `tracker.passenger` doesn’t equal `entityPlayer`

Reply:
Why should the player not receive updates of the entity they are riding? Is this already handled somewhere else?

Reply:
Oh I'm dumb, I thought tracker.passenger was the player receiving the updates and they wouldn't need to receive updates of themselves.